### DISCUSSION TITLE
Support for specifying listening IP address

### DISCUSSION THREAD

**Discussion by kayavila** [idyll-lang/idyll#777] [Q&A] [answered]
I see that --port can be used to specify the listening port, but does idyll support specifying the IP address?  I have a virtual machine with multiple interfaces and it's using the one with the gateway, which is NAT'd, not the one that's accessible to the host machine.  I'm not finding any information or documentation on how IP can be specified.  Thanks.

**Comment by mathisonian** [comment, marked as answer]
Hi @kayavila,

We don't currently support that, however it is probably possibly to add. We use `browersync` internally to handle serving the files during development. If there's a [browsersync option](https://browsersync.io/docs/options) that would fit your needs we can add support to pass that through. 

For deployment, idyll generates static HTML,CSS, and JS as output so you can host this however you like.

**Reply by kayavila** [reply]
Thanks for the response - very helpful.  It's not a big deal because I can work around it with iptables or rinetd.  It looks like the `host` or `listen` options on browersync would potentially do the trick, though.
